pkg/operator/ceph/object/zone/controller.go

```diff
@@ -230,7 +230,7 @@ func (r *ReconcileObjectZone) createCephZone(zone *cephv1.CephObjectZone, realmN
 	}
 
 	// create zone
-	_, err = object.RunAdminCommandNoMultisite(objContext, "zone", "get", realmArg, zoneGroupArg, zoneArg)
+	_, err = object.RunAdminCommandNoMultisite(objContext, false, "zone", "get", realmArg, zoneGroupArg, zoneArg)
```

Comment:
Even though the result is ignored, how about still passing `true` whenever we have a `get`.
```suggestion
	_, err = object.RunAdminCommandNoMultisite(objContext, true, "zone", "get", realmArg, zoneGroupArg, zoneArg)
```

Reply:
OK, fixed.